molecule/default/tests/test_default.py

```diff
@@ -1,20 +1,60 @@
 import os
 import re
-
+import pytest
 import testinfra.utils.ansible_runner
 
+###############################################################
+# ASC-187: Verify nova_force_config_drive setting is disabled  #
+################################################################
+# RPC 10+ manual test 1
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('compute-infra_hosts')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts(
+    ['nova_scheduler', 'nova_compute'])
 
 
-# RPC 10+ manual test 1
-def test_nova_force_config_drive_is_disabled(host):
+def test_nova_force_config_drive_is_disabled_on_nova_compute(host):
+    verify_nova_force_config_drive_is_disabled(host)
+
+
+def test_nova_force_config_drive_is_disabled_on_nova_scheduler(host):
+    verify_nova_force_config_drive_is_disabled(host)
+
+
+def verify_nova_force_config_drive_is_disabled(checked_host):
+    """Test to verify that force config_drive is disabled in checked_host.
+
+        Args:
+            checked_host(string): A hostname in dynamic_inventory.json
```

Comment:
From the perspective of the test, a host value would come from `molecule.yml` not `dynamic_inventory.json`, right?

Reply:
yes, it comes from molecule.yml, but molecule.yml is created by moleculerizy.py, originally from dynamic_inventory.json. I think dynamic_inventory.json is known crossed teams, but I have no problem changing it to molecule.yml